spec/Section 6 -- Execution.md

```diff
@@ -565,6 +555,30 @@ DoesFragmentTypeApply(objectType, fragmentType):
 Note: The steps in {CollectFields()} evaluating the `@skip` and `@include`
 directives may be applied in either order since they apply commutatively.
 
+### Root Field Collection
+
+Root field collection processes the operation's top-level selection set:
+
+CollectRootFields(rootType, operationSelectionSet, variableValues):
+
+- Initialize {groupedFieldSet} to an empty ordered map of lists.
+- Call {CollectFields(rootType, operationSelectionSet, groupedFieldSet)}.
+- Return {groupedFieldSet}.
+
+### Object Subfield Collection
+
+Root field collection processes the operation's top-level selection set:
```

Comment:
This isn't used just for root field collection, but for any object with subfields.

Reply:
Yes, incorrect. 2a120b7